config/redirects.yml

```diff
@@ -266,7 +266,21 @@ redirects:
   "/are-you-ready-to-apply": "/steps-to-become-a-teacher"
   "/whats-your-learning-style": "/ways-to-train"
   "/overseas-applicants": "/international-candidates"
-
+  "/life-as-a-teacher/teaching-vacancies-service": "/my-story-into-teaching"
+  "/five-things-every-prospective-teacher-needs-to-know": "/three-things-to-help-you-get-into-teaching"
+  "/funding-and-salary/overview/scholarships/physics-scholarships": "/salaries-and-benefits"
+  "/explore-my-options/return-to-teaching/return-to-teaching-in-england-from-overseas": "/international-returners"
+  "/regional-make-a-difference-in-bath": ""
```

Comment:
```suggestion
  "/regional-make-a-difference-in-bath": "/"
```

Reply:
Not sure if redirecting to "" will just put them at the root of the app, the suggestion might be pointless.

Reply:
Ah I'm not sure either that was just one I missed 😄